src/main/java/graphql/execution/batched/DataFetchingException.java

```diff
@@ -0,0 +1,26 @@
+package graphql.execution.batched;
```

Comment:
There is already a `graphql.ExceptionWhileDataFetching` exception.  I dont think we need another one one with such a similar name

It could be refactored to take an error message say.

or create a new BatchedDataFetcherException since this error is BatchedExecutionStrategy specific